utility/test_repoutil.py

```diff
@@ -0,0 +1,138 @@
+# Copyright 2019 Cisco and its affiliates
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+
+__author__ = "Stanislav Chlebec"
+__copyright__ = "Copyright 2019 Cisco and its affiliates, Copyright The IETF Trust 2019, All Rights Reserved"
+__license__ = "Apache License, Version 2.0"
+__email__ = "stanislav.chlebec@pantheon.tech"
+
+import unittest
+import repoutil
+from repoutil import RepoUtil
+import os
+
+test_repo_dir = '~/work/yang'
+
+
+
+
+class TestRepoutil(unittest.TestCase):
```

Comment:
I am not going to comment each test case since this is still WIP. But this is how I expected it. I ll comment once you are done and cleaned up the code if there is something to comment. 

Thank you